e2e/interchaintestv8/relayer_test.go

```diff
@@ -889,6 +890,10 @@ func (s *RelayerTestSuite) Test_10_TimeoutPacketFromCosmos() {
 func (s *RelayerTestSuite) ICS20TimeoutFromCosmosTimeoutTest(
 	ctx context.Context, proofType operator.SupportedProofType, numOfTransfers int,
 ) {
+	if os.Getenv(testvalues.EnvKeyEthTestnetType) != testvalues.EthTestnetTypePoS {
+		s.T().Skip("Skipping timeout packet test on non-PoS chain")
+	}
+
```

Comment:
Please don't skip the entire test, only skip the replay subtest at the end. And add a comment for why we are skipping in the non-POS mode